src/unix/nto/mod.rs

```diff
@@ -877,7 +877,7 @@ pub const MS_SYNC: c_int = 2;
 pub const SCM_RIGHTS: c_int = 0x01;
 pub const SCM_TIMESTAMP: c_int = 0x02;
 cfg_if! {
-    if #[cfg(not(target_env = "nto71_iosock"))] {
+    if #[cfg(not(any(target_env = "nto71_iosock", target_env = "nto80")))] {
```

Comment:
Are these not available on 8.0 either?

If this change is related to making tests pass on 8.0 rather than as part of the BPF, would you be able to put it in a separate commit?

Reply:
I still don't follow the connection - why are sysctl things like `HW_IOSTATS` related to BPF?